[0m17:32:41.366695 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000258C06999C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000258C151DE70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000258C151DC60>]}


============================== 17:32:41.380356 | 41c72dc7-9399-477b-9fd3-d89886a7a27d ==============================
[0m17:32:41.380356 [info ] [MainThread]: Running with dbt=1.9.4
[0m17:32:41.382361 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\Shreyash\\.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init sentiment_analysis', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m17:32:41.397987 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object.. Dictionary: {'dir': WindowsPath('C:/Users/Shreyash/.dbt')}
[0m17:32:41.397987 [info ] [MainThread]: Creating dbt configuration folder at 
[0m17:32:41.397987 [debug] [MainThread]: Starter project path: C:\Users\Shreyash\Desktop\Github\data_pipeline_Project\pipeline_project_venv\lib\site-packages\dbt\include\starter_project
[0m17:32:41.445560 [info ] [MainThread]: 
Your new dbt project "sentiment_analysis" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m17:32:41.445560 [info ] [MainThread]: Setting up your profile.
[0m17:36:06.101850 [info ] [MainThread]: Profile sentiment_analysis written to C:\Users\Shreyash\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m17:36:06.101850 [debug] [MainThread]: Command `dbt init` succeeded at 17:36:06.101850 after 204.79 seconds
[0m17:36:06.101850 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000258C06999C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000258C15814E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000258C1670C40>]}
[0m17:36:06.101850 [debug] [MainThread]: Flushing usage events
[0m17:36:06.262099 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m18:12:26.052832 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ACB13059C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ACB21B5FC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ACB21B5DE0>]}


============================== 18:12:26.055833 | 2743327d-85c6-4885-9968-ef1b9357d3d0 ==============================
[0m18:12:26.055833 [info ] [MainThread]: Running with dbt=1.9.4
[0m18:12:26.057336 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Shreyash\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt run', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m18:12:26.058341 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\Shreyash\Desktop\Github\data_pipeline_Project\dbt_project\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m18:12:26.059349 [debug] [MainThread]: Command `dbt run` failed at 18:12:26.058341 after 0.07 seconds
[0m18:12:26.059349 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ACB13059C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ACB21B5DE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001ACB21B6080>]}
[0m18:12:26.059349 [debug] [MainThread]: Flushing usage events
[0m18:12:26.160610 [debug] [MainThread]: An error was encountered while trying to flush usage events
